src/binary/ibc-setup/commands/ics20.ts

```diff
@@ -142,7 +156,13 @@ async function resolveLink(
     return link;
   }
 
-  const link = await Link.createWithNewConnections(nodeA, nodeB);
+  const link = await Link.createWithNewConnections(
+    nodeA,
+    nodeB,
+    logger,
+    srcTrust ?? undefined,
```

Comment:
That is one way. Feel free to make anything that takes `T | undefined` take `T | undefined | null` if that is better for you.

Reply:
e4b7091